main.go

```diff
@@ -410,13 +412,28 @@ func newUser(s ssh.Session) *User {
 
 	Log.Println("Connected " + u.Name + " [" + u.id + "]")
 
-	if bansContains(Bans, u.addr, u.id) || TORIPs[u.addr] {
-		Log.Println("Rejected " + u.Name + " [" + host + "] (banned)")
-		u.writeln(Devbot, "**You are banned**. If you feel this was a mistake, please reach out to the server admin. Include the following information: [ID "+u.id+"]")
+	if TORIPs[u.addr] {
+		Log.Println("Rejected " + u.Name + " [" + host + "] (Tor IP)")
+		u.writeln(Devbot, "**You are not allowed to join as you are using a Tor IP**")
 		s.Close()
 		return nil
 	}
 
+	banInfo := getBan(Bans, u.addr, u.id)
+	if banInfo != nil {
+		if banInfo.UseTime && banInfo.UnbanTime.Before(time.Now()) {
+			unbanIDorIP(banInfo.ID)
+		} else {
+			Log.Println("Rejected " + u.Name + " [" + host + "] (banned)")
+			u.writeln(Devbot, "**You are banned**. If you feel this was a mistake, please reach out to the server admin. Include the following information: [ID "+u.id+"]")
+			if banInfo.UseTime {
+				u.writeln(Devbot, "You will be unbaned on "+banInfo.UnbanTime.Format(time.RFC3339))
```

Comment:
use a duration here instead of an absolute time

Reply:
Done!

Reply:
It looks like that:
```
devbot: You will be unbaned in 41 days, 15 hours, 56 minutes, and 8 seconds.
```
or like that:
```
devbot: You will be unbaned in 13 seconds.
```

Reply:
It's quite verbose. Which could be fine. But we do also have a pretty duration printer here: https://github.com/Arkaeriit/devzat/blob/c98c58e03fd3703f5d500424e886e5f06fe0768d/util.go#L420

what do you think?

Reply:
Ho right, I didn't remember this function. `printPrettyDuration` is not very suitable for duration longer than a day, so not great for bans.
But maybe we can merge `printPrettyDuration` and `formatDuration` to prevent code duplication.
I'll make a proposition of it.

Reply:
I went with full unit names. So it changes the way the relative time display is. If you don't like it, I will go back to units symbols.
![image](https://github.com/user-attachments/assets/f50ef090-98ec-479f-ab8a-42fd0964da5a)

Reply:
I see what you mean. maybe it's better to keep it short but add ability to display days using the unit 'd'

Reply:
I reverted back to using 1 letter units with no space between the numeric value and the unit.

Reply:
It looks like that now: 
![image](https://github.com/user-attachments/assets/b5b3ca4e-a242-474a-8afc-0597185805b3)